ontology/ontology.go

```diff
@@ -10,10 +10,12 @@ import (
 	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/gtf"
 	"github.com/vertgenlab/gonomics/interval"
-	"github.com/vertgenlab/gonomics/numbers"
+	//"github.com/vertgenlab/gonomics/interval/lift"
+	//"github.com/vertgenlab/gonomics/numbers"
 	"github.com/vertgenlab/gonomics/ontology/gaf"
 	"github.com/vertgenlab/gonomics/ontology/obo"
 	"log"
+	"math"
 	"strings"
 )
 
```

Comment:
genesAssignmentsFromGaf needs to be able to take genes listed in the original obo as well, or we need to exclusively use the common gaf method that I came up with as a temporary fix